Formula/php.rb

```diff
@@ -20,15 +20,18 @@ class Php < Formula
   depends_on "curl-openssl"
   depends_on "freetds"
   depends_on "freetype"
+  depends_on "gd"
```

Comment:
Isn't this part of macOS? Why is this needed if we had the option enabled before?

Reply:
To quote:

> `--with-gd` no longer exists. Use `--enable-gd` to enable the extension and `--with-external-gd` to use Homebrew's gd rather than the bundled one (unless Homebrew's one is incompatible).

I've made a small mistake previously where I used `--with-gd` instead of `--with-external-gd`. I've fixed and tested locally and everything builds successfully. I can push that change, but I can also drop the dependency and use the bundled one instead.

Reply:
I'd say the bundled one might cause less surprise bugs. Let's do that one.

Reply:
Yeah both the bundled and external work. If the bundled version is linked statically then you can use either or - not sure if there's a preference PHP has or not. I think we don't use external `pcre` so feel free to drop the option.

Also note: the option also doesn't take in a directory so you don't need to do that.

Reply:
If we use the bundled GD, we need to specify which features it should support.

```
  --enable-gd             Include GD support
  --with-external-gd      Use external libgd
  --with-webp             GD: Enable WEBP support (only for bundled libgd)
  --with-jpeg             GD: Enable JPEG support (only for bundled libgd)
  --with-xpm              GD: Enable XPM support (only for bundled libgd)
  --with-freetype         GD: Enable FreeType 2 support (only for bundled
                          libgd)
  --enable-gd-jis-conv    GD: Enable JIS-mapped Japanese font support (only
                          for bundled libgd)
```

Reply:
I switched to bundled gd to maintain the status quo. Switching to external gd is best taken care of in a separate PR if maintainers agree that it should be done.